### ISSUE TITLE
onnx2ncnn issue with onnx (opset_version=14 )

### ISSUE THREAD

**Issue by harryporter2023** [Tencent/ncnn#4450]
## model | 模型 | モデル
1. onnx optset_version 14

## how to reproduce | 复现步骤 | 再現方法
When onnx is exported as optset_version 9, the softmax attribute axis value is 2.
When onnx is exported as optset_version 14, the softmax attribute axis value is -1.
**The  ncnn lib cannot run when softmax attribute axis value is -1, is there any solution?**

**onnx optset_version= 9**

![image](https://user-images.githubusercontent.com/122018487/210763211-74c0839c-3c2e-44f6-94d7-ee8f6d10c1d5.png)

![image](https://user-images.githubusercontent.com/122018487/210762854-ff4a3ed2-105b-419d-a66a-495845dc3d47.png)

**onnx optset_version= 14**

![image](https://user-images.githubusercontent.com/122018487/210763117-25721ca2-2de3-42b4-9e48-58e68c0d7a39.png)

![image](https://user-images.githubusercontent.com/122018487/210762667-683b3ba1-3318-4416-93de-d2307ac86493.png)

**Comment by nihui** [comment]
针对onnx模型转换的各种问题，推荐使用最新的pnnx工具转换到ncnn
In view of various problems in onnx model conversion, it is recommended to use the latest pnnx tool to convert your model to ncnn

```shell
pip install pnnx
pnnx model.onnx inputshape=[1,3,224,224]
```

详细参考文档
Detailed reference documentation
https://github.com/pnnx/pnnx
https://github.com/Tencent/ncnn/wiki/use-ncnn-with-pytorch-or-onnx#how-to-use-pnnx
